test-cases/tests/monitoring/m03-verify-number-of-pods-for-amo.md

```diff
@@ -0,0 +1,17 @@
+---
+tags:
+  - 2.4.0
+---
+
+# M03 - Verify number of pods for AMO
```

Comment:
@jhellar This one may not land for 2.4 freeze. It doesn't have an assignee yet

Reply:
I've marked the test case as targeting 2.5.0.